src/win/core.c

```diff
@@ -80,6 +81,38 @@ static void uv__crt_invalid_parameter_handler(const wchar_t* expression,
 }
 #endif
 
+static QUEUE uv_loops;
+static uv_mutex_t uv_loops_lock;
+
+static void uv__loops_init() {
+  uv_mutex_init(&uv_loops_lock);
+  QUEUE_INIT(&uv_loops);
+}
+
+static void uv__loops_add(uv_loop_t* loop) {
+  uv_mutex_lock(&uv_loops_lock);
+  QUEUE_INIT(&loop->uv_loops);
+  QUEUE_INSERT_TAIL(&uv_loops, &loop->uv_loops);
+  uv_mutex_unlock(&uv_loops_lock);
+}
+
+static void uv__loops_remove(uv_loop_t* loop) {
+  uv_mutex_lock(&uv_loops_lock);
+  QUEUE_REMOVE(&loop->uv_loops);
+  uv_mutex_unlock(&uv_loops_lock);
+}
+
+void uv_wake_all_loops() {
```

Comment:
make this `uv__wake_all_loops` since it's an internal API. I know the Windows side doesn't quite follow this convention, but we try to do it for new code.